packages/ember-toucan-core/src/components/form/fields/file-input.ts

```diff
@@ -15,25 +18,32 @@ interface ToucanFormFileInputFieldComponentSignature {
     error?: ErrorMessage;
     files?: File[];
     hint?: string;
-    label: string;
+    label?: string;
     trigger: string;
     isDisabled?: boolean;
     multiple?: boolean;
     onChange?: (files: File[], event: FileEvent) => void;
     rootTestSelector?: string;
```

Comment:
As an aside / not part of this PR: it may be good to add JSDocs here like with our other components so that they show up in your editor!